armsrc/iso14443a.c

```diff
@@ -1926,24 +1927,37 @@ void ReaderIso14443a(UsbCommand *c)
 	}
 
 	if(param & ISO14A_CONNECT) {
+		LED_A_ON();
+		clear_trace();
```

Comment:
Adding clear_trace() here prevents tracing of sequences with several selects (hf mf mifare, hf mf nested).

Reply:
Here ill add flag to clear and raise it from apdu